django/db/migrations/autodetector.py

```diff
@@ -919,6 +939,46 @@ def generate_altered_fields(self):
                     self._generate_removed_field(app_label, model_name, field_name)
                     self._generate_added_field(app_label, model_name, field_name)
 
+    def create_altered_indexes(self):
+        option_name = 'indexes'
+        for app_label, model_name in sorted(self.kept_model_keys):
+            old_model_name = self.renamed_models.get((app_label, model_name), model_name)
+            old_model_state = self.from_state.models[app_label, old_model_name]
+            new_model_state = self.to_state.models[app_label, model_name]
+
+            old_indexes = old_model_state.options.get(option_name)
```

Comment:
Can you use `['indexes']` here? If not, the list comprehensions in the next lines have a `not in None` and will fail.